BAZProjectC1/Extensions/UIImageViewExtension.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  UIImageViewExtension.swift
+//  BAZProjectC1
+//
+//  Created by 1044336 on 17/09/22.
+//
+
+import Foundation
+import UIKit
+extension UIImageView{
+    public func loadImageFromUrl(urlString: String){
+        if let url = URL(string: urlString) {
+            DispatchQueue.global().async { [weak self] in
+                if let data = try? Data(contentsOf: url){
+                    if let image = UIImage(data: data){
```

Comment:
Podemos agrupar estas validaciones con un guard let?
```suggestion
                guard let data = try? Data(contentsOf: url), let image = UIImage(data: data) else { return }
```